hack/make-rules/test-cmd-util.sh

```diff
@@ -5341,6 +5341,9 @@ runTests() {
     kube::test::get_object_assert 'clusterrolebindings -l test-cmd=auth' "{{range.items}}{{$id_field}}:{{end}}" 'testing-CRB:'
     kube::test::get_object_assert 'clusterroles -l test-cmd=auth' "{{range.items}}{{$id_field}}:{{end}}" 'testing-CR:'
 
+    failure_message=$(! kubectl auth reconcile "${kube_flags[@]}" -f test/fixtures/pkg/kubectl/cmd/auth/rbac-v1beta1.yaml 2>&1 )
```

Comment:
Can we put `test-cmd-util.sh` into a directory owned by sig-cli?

For example, one could create a `hack/make-rules/cmd` with its own `OWNERS` file that tags sig-cli people as reviewers/approvers

Reply:
> Can we put test-cmd-util.sh into a directory owned by sig-cli?
> 
> For example, one could create a hack/make-rules/cmd with its own OWNERS file that tags sig-cli people as reviewers/approvers

I'm fine with that.  @fejta you really want that done here or can we proceed with that separately?

Reply:
@fejta the topic of moving/cleaning this is a separate issue, we're planning to clean this file, since this is hard to maintain and work with it. Ideally this will be split into a separate smaller chunks. But that's a larger task we're hoping to tackle soon-ish.

Reply:
> Ideally here, so that I can approve this PR with an OWNERS file that delegates subsequent changes to sig-cli.
> 
> Although apparently this is already approved

I happen to be hack approver, so it was approved via pull author powers.